### PR TITLE
Add example10 to Top Level README

### PR DESCRIPTION
**Pull request by cmoussa1**
This is a tiny PR that just adds _example10_, a hyperlink to its directory in `master`, and a description of the example in the top level **README.md**.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,7 @@ Use KVS Python interfaces to store user data into KVS
 **_9. [A Data Conduit Strategy](https://github.com/flux-framework/flux-workflow-examples/tree/master/example9)_**
 
 Attach to a job that receives OS time data from compute jobs
+
+**_10. [Python Job Submit/Wait](https://github.com/cmoussa1/flux-workflow-examples/tree/master/example10)_**
```
Comment (SteVwonder) [inline README.md]:
```suggestion
**_10. [Python Job Submit/Wait](https://github.com/flux-framework/flux-workflow-examples/tree/master/example10)_**
```

Comment (cmoussa1) [reply README.md]: 🤦‍♂️ thanks for catching that! I went ahead and made that change.

**Comment by dongahn** [comment]
LGTM.

**Comment by cmoussa1** [comment]
Thanks!

**Pull request merged**